app/models/token/expirable_token.rb

```diff
@@ -64,30 +64,34 @@ def valid_plaintext?(input)
       end
 
       def expired?
-        expires_on.nil? || Time.now > expires_on
+        expires_on.nil? || Time.current > expires_on
```

Comment:
I don't think that there's a difference here between `Time.now` and `Time.current`, because the comparison happens with the TZ taken into account. But since the question is whether `expires_on` has passed or not yet, what do you think about:

```suggestion
        expires_on.nil? || expires_on.past?
```

Reply:
Works for me, this line changed only because rubocop complained about it